ee/clickhouse/client.py

```diff
@@ -166,6 +170,15 @@ def sync_execute(query, args=None, settings=None, with_column_types=False):
                     save_query(prepared_sql, execution_time)
         return result
 
+    def _convert_to_dict(result) -> List[Dict[str, Any]]:
```

Comment:
This is pretty incomprehensible without better naming. Perhaps extract `header_row = result[1]` or smth?